### DISCUSSION TITLE
没有收到激活码

### DISCUSSION THREAD

**Discussion by weizijian** [getlantern/lantern-forum-cn#579] [Q&A]
6月13号 买了两年的专业版 没有收到激活码 893586239@qq.com

**Comment by MaxMKMax** [comment]
@weizijian 您好. 请您也提供我支付单据. 谢谢!
我帮您提交问题. 请给我们几天工作日时间. 尽快回复您. 请您之后留意这个帖子或邮箱. 谢谢您的耐心等待! 也非常感谢您支持蓝灯!

**Comment by MaxMKMax** [comment]
@weizijian 您好. 请您也提供我支付单据. 谢谢!

**Comment by weizijian** [comment]
@Mike777Khoo  你看下这两个可以吗
<img width="439" alt="微信图片_20240620144159" src="https://github.com/getlantern/lantern-forum-cn/assets/12810520/f3c8b0f7-dc0e-48fc-8002-065397ff64a3">
![微信图片_20240620144206](https://github.com/getlantern/lantern-forum-cn/assets/12810520/6b5ef9ac-a5bf-43bc-8f9b-8103ff5cf7aa)

**Comment by MaxMKMax** [comment]
@weizijian https://lantern.surveysparrow.com/s/propayment-triage-form/tt-ZxBeJ
您好. 为了保护您的隐私. 请您发支付单据和账号邮箱到这个链接. 谢谢.
